iis/mymodule.cpp

```diff
@@ -835,19 +831,8 @@ CMyHttpModule::OnBeginRequest(
 			goto Finished;
 		}
 
-		WIN32_FILE_ATTRIBUTE_DATA fdata;
-		BOOL ret;
-
-		ret = GetFileAttributesEx(path, GetFileExInfoStandard, &fdata);
-
-		pConfig->m_dwLastCheck = ctime;
-
-		if(pConfig->m_Config == NULL || (ret != 0 && (pConfig->m_LastChange.dwLowDateTime != fdata.ftLastWriteTime.dwLowDateTime ||
-			pConfig->m_LastChange.dwHighDateTime != fdata.ftLastWriteTime.dwHighDateTime)))
+		if(pConfig->m_Config == NULL)
```

Comment:
Why do we need this check again? Seems identical to the check on line 821? Also, doesn't seem like the `pConfig` state should change between 821 and 834?

Reply:
another thread could change it

Reply:
ignore the above. I thought it was relating to a different places.
Yes, you are right, this check is not needed. Removing.